_config.yml

```diff
@@ -32,33 +32,33 @@ exclude:
     "_docs/cleanup.sh",
   ]
 navbar_items:
-  - name: "Assignments"
-    link: "/assignments.html"
+  - name: "Staff"
```

Comment:
Why were these links reordered? It makes sense for assignments to be first since this page would mainly be frequented by students looking for MP/lab docs

Reply:
Fair enough.